components/book/BookCard/BookCard.tsx

```diff
@@ -1,19 +1,37 @@
 import React from 'react'
 import Image from 'next/image'
-import { Book } from '../../../types'
+import Link from 'next/link'
+
+import { Book } from 'types'
+import { formatPrice } from 'utils/formatPrice'
+
+import styles from './BookCard.module.scss'
 
 const BookCard = ({ book }: { book: Book; eager: boolean }) => {
+  const {
+    title,
+    user: {
+      district: { name: districtName },
+    },
+    price,
+    pictures: [firstPicture],
```

Comment:
`pictures `look like an object, can we named it to something meaningful like `pictureUrls `

Reply:
I think `pictures` is fine. 
- We use typescript so a picture has type string.
- We may want to save more info for a picture in the future. Once updating a field name, migration happens and we both know this is costly.
- Guess this is not so confusing since a picture has a type string received from the server. What else we can guess except for an URL?
- Auth0 uses the word picture as an image URL so I think this abbreviation is common. [Action - Sync users](https://manage.auth0.com/dashboard/us/dev-efbkdivge87ynzb6/actions/library/details/ab44a690-feac-43b8-a0cb-dc42af92a64a)
Is that make sense? 😸

Reply:
thanks, it's ok to me